.gitattributes

```diff
@@ -0,0 +1,7 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
```

Comment:
what is the purpose of this file?

Reply:
To associate attributes with certain files. For eg. we can tell git how to handle line endings or we can define merge strategies used during a branch merge